packages/OreIdWalletConnect/src/OreIDWalletConnect/OreIDWalletConnect.tsx

```diff
@@ -83,8 +83,7 @@ export const OreIDWalletConnect: React.FC<OreIDWalletConnectProps> = ({
   const onConnectionCreate: WalletConnectRefEvent = (connection, payload) => {
     if (props.onConnectionCreate) {
       try {
-        const currentConnection = getCurrentConnectionByUri(connection.connector.uri)
```

Comment:
Don't do this, you will get an error here. The variable received in connection may be out of date. The updated connection is always/only the one in useRef.

This code will insert bugs.